views/partials/search-form-fields.html

```diff
@@ -24,12 +24,11 @@
 
   {% if trials %}
     <ul class="breadcrumb">
-      <li>
-        <a href="/search">Search</a>
-      </li>
-      <li>
-        Found {{ trials.total_count | formatNumber }} trial{% if trials.total_count != 1 %}s{% endif %}
-      </li>
+      <li><a href="/">Home</a></li>
+      <li><a>Search</a></li>
+      {% if query.q %}
+        <li><a>{{ query.q }}</a></li>
```

Comment:
Add the number of trials as well. Maybe something like `{{ query.q }}({{ trials.total_count | formatNumber }} trial(s))`

Also, make sure the breadcrumb for the current page isn't clickable